src/NServiceBus.Core/Transports/Msmq/MsmqConfigurationExtensions.cs

```diff
@@ -60,5 +60,14 @@ public static InstanceMappingFileSettings InstanceMappingFile(this RoutingSettin
         {
             return new InstanceMappingFileSettings(config.Settings);
         }
+
+        /// <summary>
+        /// Makes sure that the DLQ flag is set on messages that have time to reach queue set.
```

Comment:
> Makes sure that the DLQ flag is set on messages that have time to reach queue set.

how about:

> Moves expired messages using the "time to reach queue" setting to the dead letter queue instead of discarding them.